core/src/main/java/edu/wpi/grip/core/operations/composite/RGBThresholdOperation.java

```diff
@@ -15,7 +15,7 @@
  */
 public class RGBThresholdOperation extends ThresholdOperation {
 
-    private static final Logger logger =  Logger.getLogger(RGBThresholdOperation.class.getName());
+    private static final Logger logger = Logger.getLogger(RGBThresholdOperation.class.getName());
```

Comment:
Because autoformat (there was an extra space before, IntelliJ automatically removed it)